### PR TITLE
Cloudbase init switch

### PR DESCRIPTION
**Pull request by ilenghel**
Added a switch parameter to select which Cloudbase-Init branch to use.

### CODE REVIEW

File: WinImageBuilder.psm1 [hunk 1]
```diff
@@ -351,14 +351,25 @@ function Download-CloudbaseInit {
         [Parameter(Mandatory=$true)]
         [string]$resourcesDir,
         [Parameter(Mandatory=$true)]
-        [string]$osArch
+        [string]$osArch,
+        [parameter(Mandatory=$false)]
+        [switch]$develBranch=$true
```
Comment (costingalan) [inline WinImageBuilder.psm1]: I would change from `$develBranch` to something like `$betaRelease`

File: WinImageBuilder.psm1 [hunk 2]
```diff
@@ -351,14 +351,25 @@ function Download-CloudbaseInit {
         [Parameter(Mandatory=$true)]
         [string]$resourcesDir,
         [Parameter(Mandatory=$true)]
-        [string]$osArch
+        [string]$osArch,
+        [parameter(Mandatory=$false)]
+        [switch]$develBranch=$true
+
     )
     Write-Host "Downloading Cloudbase-Init..."
```
Comment (costingalan) [inline WinImageBuilder.psm1]: I would also believe a better way to address the Cloudbase-Init installation is to write a function for this.

File: WinImageBuilder.psm1 (hunk 1 above)
Comment (costingalan) [inline WinImageBuilder.psm1]: I would change `$develBranch` to something like `$betaRelease`

**Review by costingalan — CHANGES_REQUESTED** [review]

**Comment by costingalan** [comment]
Also please do a `git rebase` in order to have only one commit in the end.

**Comment by ilenghel** [comment]
Wrong branch submitted. Closing PR.

**Pull request closed without merging**